docs/reference/run.md

```diff
@@ -951,6 +952,21 @@ If you do block IO in the two containers at the same time, by, for example:
 You'll find that the proportion of time is the same as the proportion of blkio
 weights of the two containers.
 
+The --blkio-weight-device="DEVICE_NAME:WEIGHT" flag sets a specific device weight.
+The DEVICE_NAME:WEIGHT is a string containing a colon-separated device name and weight.
+For example, to set /dev/sda device weight to 200:
+
+$ docker run -it \
+    --blkio-weight-device "/dev/sda:200" \
+    ubuntu
+
+If you specify both the --blkio-weight and --blkio-weight-device, Docker uses the --blkio-weight as the default weight and uses --blkio-weight-device to override this default with a new value on a specific device. The following example uses a default weight of 300 and overrides this default on dev/sda setting that weight to 200:
```

Comment:
Can you;

- wrap this line to 80 characters
- add backticks around the flags and `dev/sda` to format them as code?